### ISSUE TITLE
How are events from Meetup collected now?

### ISSUE THREAD

**Issue by szabgab** [rust-lang/this-week-in-rust#6182]
It would be nice to know how you collect it or which groups you are monitoring to know if I should send a PR adding one of my events or if I should rely on the manual (or automated ?  #6008 ) process to collect it. 
Especially given that I have two Meetup groups. One is dedicated to Rust, but in the other one I have other, non-Rust events as well.

**Comment by bdillo** [comment]
We have a bit of automation for meetups that needs to get cleaned up before we start using it more, so right now for meetup we are just manually checking a list of known groups and occasionally searching for new groups. Can you link the meetup groups here? I'll add them to our list to check each week and we will make sure any events that fall within the date range for the week will get included. We do check a few non-specific rust groups and pick out rust events, so that shouldn't be a big issue for your non-rust specific group.

**Comment by szabgab** [comment]
The two groups I run are:

https://www.meetup.com/rust-tlv/ Rust Tel Aviv (fully Rust, presentation are usually in Hebrew)
https://www.meetup.com/code-mavens/   Code Mavens (partially Rust, presentations are virtual and in English)

Thanks

**Comment by szabgab** [comment]
Would it be possible to publish the list of groups you are monitoring? (e.g. add it as a file to this repo)

**Comment by bdillo** [comment]
https://github.com/rust-lang/this-week-in-rust/blob/master/tools/events/meetups.json

It's there right now but needs to get updated, I have another copy in use so will probably put that here when I look at the meetup scripts again. I'll leave this issue open until I add your groups there.

**Comment by mariannegoldin** [comment]
+1 to what @bdillo said: we are using the `meetups.json` as a starting point for automation, while also relying on the calendar, PRs, and manual searches, as well as word-of-mouth ad hoc additions.

TLDR: If your meetup group is on the list (which they are), you should be good to go. However, feel free to post a PR with any new events, we appreciate it!

**Comment by mariannegoldin** [comment]
Confirming that the two groups above have been added to `meetups.json`: closing this issue.

**Comment by szabgab** [comment]
I see also the events were added the https://github.com/rust-lang/this-week-in-rust/blob/master/content/2024-12-18-this-week-in-rust.md

Thank you.

BTW As I prepared the "social report" I went over the list in that json file and found that several of the groups don't exist (any more) and that there are multiple links actually linking to the same group.

In Meetup you can change the url and they will show your group on both urls. Instead of redirecting the old one to the new which makes it a tad harder to notice when to addresses are actually the same group.
